less/availity-cards.less

```diff
@@ -63,6 +91,29 @@
   }
 }
 
+.card-header-link {
```

Comment:
I added `.card-header-link` and `.card-title-link` which basically do the same thing.  would that be sufficient?